### DISCUSSION TITLE
Embed Component: Twitter/Instagram Content

### DISCUSSION THREAD

**Discussion by jennifervelez08** [adobe/aem-core-wcm-components#1520] [Q&A]
Does the Embed component support Instagram/Twitter content out of the box? Is customization required to create live Instagram/Twitter feeds for landing pages?

**Comment by vladbailescu** [comment]
The Embed component supports [Instagram](https://github.com/adobe/aem-core-wcm-components/blob/master/config/src/content/jcr_root/apps/core/wcm/config/com.adobe.cq.wcm.core.components.internal.services.embed.OEmbedClientImplConfigurationFactory-instagram.config) and [Twitter](https://github.com/adobe/aem-core-wcm-components/blob/master/config/src/content/jcr_root/apps/core/wcm/config/com.adobe.cq.wcm.core.components.internal.services.embed.OEmbedClientImplConfigurationFactory-twitter.config) through [OEMBED](https://oembed.com/) (I linked each service to the respective OEMBED configuration). You might need to tweak or add configurations if that does not cover your usage. See an example of an embedded tweet at https://aemcomponents.dev/content/core-components-examples/library/page-authoring/embed.html#title-f67fda4f48
